source/_integrations/nest.markdown

```diff
@@ -26,6 +26,12 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
+<div class='note warning'>
+<p>The instructions on this page do not work for new credentials due to a change in Google's desktop authentication. Follow the instructions in <a href="https://github.com/home-assistant/core/issues/67662#issuecomment-1079977875">this GitHub comment</a> for a workaround.</p>
```

Comment:
Maybe instead of "Follow the instructions in ..." say something like to see that issue for more deatils? I think its not necessarily a workaround everyone can follow